_docs/_latest/develop/test/integration-testing/run-test-suite.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Run The Test Suite Locally
+order: 045
+---
+
+# {{ page.title }}
+
+This page describes how to run the integration test suite for your Forseti
+contributions on your local dev environment. 
+
+## **Setting up development environment**
+
+- Create a new project in your organization.
+- Terraform uses an IAM Service Account to deploy and configure resources on 
+behalf of the user. You can create the Service account, and grant the necessary 
+roles to the Service Account manually or by running a helper script. 
+
+### **Creating Service Account by running the helper script**
+
+In order to execute this script, you must have an account with the following 
+list of permissions: 
+* resourcemanager.organizations.list
+* resourcemanager.projects.list
+* billing.accounts.list
+* iam.serviceAccounts.create
+* iam.serviceAccountKeys.create
+* resourcemanager.organizations.setIamPolicy
+* resourcemanager.projects.setIamPolicy
+* serviceusage.services.enable on the project
+* servicemanagement.services.bind on following services:
+  * cloudresourcemanager.googleapis.com
+  * cloudbilling.googleapis.com
+  * iam.googleapis.com
+  * admin.googleapis.com
+  * appengine.googleapis.com
+* billing.accounts.getIamPolicy on a billing account.
+* billing.accounts.setIamPolicy on a billing account.
+- `Billing Account Administrator` role should be granted to the admin account to 
+be able to create the Service Account using the helper script. You can revoke 
+this role after the service account is created.
+- To run the [helper script](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/helpers/setup-sa.sh), 
+clone the [Terraform Google Project factory repository](https://github.com/terraform-google-modules/terraform-google-project-factory)
+and run the following command:
+
+`./helpers/setup-sa.sh <ORGANIZATION_ID> <PROJECT_NAME> <BILLING_ACCOUNT>`
+
+### **Creating Service Account and granting roles manually**
+
+Alternatively, you can grant the following roles and enable the APIs manually.
+
+On the organization:
+
+* roles/resourcemanager.organizationAdmin
+* roles/iam.securityReviewer
+* roles/resourcemanager.folderViewer
+* roles/resourcemanager.organizationViewer
+* roles/resourcemanager.projectCreator
+* roles/billing.user
+
+On the project:
+
+* roles/owner
+* roles/compute.instanceAdmin
+* roles/compute.networkViewer
+* roles/compute.securityAdmin
+* roles/iam.serviceAccountAdmin
+* roles/serviceusage.serviceUsageAdmin
+* roles/iam.serviceAccountUser
+* roles/storage.admin
+* roles/cloudsql.admin
+
+Enable the following APIs on the Forseti project:
+
+* cloudresourcemanager.googleapis.com
+* compute.googleapis.com
+* serviceusage.googleapis.com
+
+On the host project when using shared VPC
```

Comment:
I don't think we need to provide instructions on using a shared VPC. The test fixtures do not support this.